upi/gcp/02_lb_ext.py

```diff
@@ -1,5 +1,6 @@
 def GenerateConfig(context):
 
+	// Refer to docs/dev/kube-apiserver-health-check.md on how to correctly setup health check probe for kube-apiserver
```

Comment:
this is probably not correct comment syntax in python

Reply:
oops, my bad. fixed.